caldp/20220214/CALDP_20220214_COS_linux_py38_final.yml

```diff
@@ -0,0 +1,118 @@
+# delivery_name: CALDP_20220214_COS_final
+# creation_time: Mon Feb 14 15:00:00 2022
+# conda_version: 4.10.3
+# condabuild_version: 3.18.11
+channels:
+  - https://ssb.stsci.edu/astroconda-staging
+  - https://ssb.stsci.edu/astroconda
+  - conda-forge
+  - http://ssb.stsci.edu/astroconda
+  - stsci
```

Comment:
"STScI" channel should not be here.
Trailing astroconda channel is not needed either

Reply:
This is interesting as this file was created by generating a Conda environment using the caldp_20211129 YML file, updating only the CALCOS package, and then writing out the a new YML file.  In any case,
`stsci` and `http://ssb.stsci.edu/astroconda` channels have been removed. 

The updated file will be delivered via another means to Brendan as I cannot paste the YML file here.